StartMenu.py

```diff
@@ -0,0 +1,355 @@
+# -*- coding: utf-8 -*-
+import pygame
+import configparser
+import json
+import re
+import os
+
+
+class ConfigHandler(object):
+    ''' Klasse behandelt alle Konfigs die geladen / gespeichert / veraendert werden
+    '''
+    Configurations = dict()
+
+    @staticmethod
+    def set(filename="standard", *args):
+        # setzt den jeweiligen Wert neu
+        config = ConfigHandler.Configurations[filename]    
+        if len(args) == 3:
+            value = json.loads(config.get(args[0], args[1]))
+            config.set(args[0], args[1], str(args[2]))
+            if value != args[2]:
+                ConfigHandler.save_file(filename)
+
+    @staticmethod
+    def load_dict(defaults, filename):
+        # Uebergebenes Dictionary muss immer zuerst eingelesen werden
+        # Konfig soll diese Standardwerte immer ueberschreiben
+        if filename not in ConfigHandler.Configurations:
+            ConfigHandler.Configurations[filename] = configparser.ConfigParser()
+            ConfigHandler.Configurations[filename].read_dict(defaults)
+            ConfigHandler.load_file(filename)
+    
+    @staticmethod
+    def save_file(filename):
+        if filename != "standard":
+            with open(filename, 'w') as configfile:
+                ConfigHandler.Configurations[filename].write(configfile)
+
+    @staticmethod
+    def load_file(filename):
+        # Datei einlesen mittels ConfigParser
+        if filename not in ConfigHandler.Configurations:
+            ConfigHandler.Configurations[filename] = configparser.ConfigParser()
+        if filename != "standard":
+            try:
+                with open(filename, 'r') as configfile:
+                    ConfigHandler.Configurations[filename].read_file(configfile)
+            except FileNotFoundError:
+                print("Datei existiert nicht, wenn Optionen geandert werden, wird diese geschrieben")
+
+    @staticmethod
+    def get_options(filename, sections):
+        ''' Example Usage
+            # (self.display, self.menu) = ConfigHandler.get_options(filename, ("display", "menu",))
+        '''
+        dict1 = dict()
+
+        ConfigHandler.load_file(filename)
+        config = ConfigHandler.Configurations[filename]
+        for section in sections:
+            dict1[section] = dict()
```

Comment:
dict1 ist für mich ein temporärer Name eines Dictionarys. In der kurzen Zeit ist mir kein besserer eingefallen.